specs/merge/fork-choice.md

```diff
@@ -115,7 +115,7 @@ def validate_merge_block(block: BeaconBlock) -> None:
     if TERMINAL_BLOCK_HASH != Hash32():
         # If `TERMINAL_BLOCK_HASH` is used as an override, the activation epoch must be reached.
         assert compute_epoch_at_slot(block.slot) >= TERMINAL_BLOCK_HASH_ACTIVATION_EPOCH
-        return block.block_hash == TERMINAL_BLOCK_HASH
+        assert block.body.execution_payload.parent_hash == TERMINAL_BLOCK_HASH
```

Comment:
```suggestion
        assert block.body.execution_payload.parent_hash == TERMINAL_BLOCK_HASH
    else:
        # ...  the rest
```

Reply:
nay! should be a `return` statement as noted by @mkalinin

Reply:
sorry, misread the PR when I so adamantly  said "nay". Both work